communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -1,9 +1,159 @@
+        An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph.
+      button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fmegahit%2Fmegahit"
+    - title_md: <code>checkm</code>
+      description_md: >
+        Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes
+        <ul>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_analyze">checkm_analyze</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_lineage_set">checkm_lineage_set</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_lineage_wf">checkm_lineage_wf</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_plot">checkm_plot</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_qa">checkm_qa</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_taxon_set">checkm_taxon_set</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_taxonomy_wf">checkm_taxonomy_wf</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_tetra">checkm_tetra</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_tree">checkm_tree</a></li>
+          <li><a href="{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fcheckm%2Fcheckm_tree_qa">checkm_tree_qa</a></li>
+        </ul>
+
+- id: interactive-tools
+  title: Interactive tools
+  heading_md: >
+    Interactive tools for microbial research 
+  content:
+    - title_md: <code>Pavian</code> - Interactive analysis of metagenomics data
+      description_md: >
+        An interactive browser application for analyzing and visualization metagenomics classification results from classifiers such as Kraken, KrakenUniq, Kraken 2, Centrifuge and MetaPhlAn. Pavian also provides an alignment viewer for validation of matches to a particular genome.
+      inputs:
+        - datatypes:
+            - Kraken report
+            - MetaPhlAn report
+      button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_pavian"
+    - title_md: <code>Phinch</code> - A framework for visualizing bio data
+      description_md: >
+        Interactive tool for visualising Biom data.
+      inputs:
+        - datatypes:
+            - Biom
+      button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_phinch"
+
+- id: databases
```

Comment:
I would put the database in the other way. The name of the database + tool in the title. 
Example:
```
title_md: Refseq for [<code>Kraken</code>]({{ galaxy_base_url }}/root?tool_id=...) and [<code>Bracken</code>]({{ galaxy_base_url }}/root?tool_id=...)
description_md: >
     Refseq indexes for <code>Kraken</code> and <code>Bracken</code> maintained by [Ben Langmead](https://benlangmead.github.io/aws-indexes/k2) 
        <ul>
          <li> Viral (Refeq viral)</li>
          <li> MinusB (Refeq archaea, viral, plasmid, human1, UniVec_Core) </li>
         ... Other lines in the table in the link above
        </ul>
```

Reply:
I did it for the first two for the last 2 I don't know what is better to put

Reply:
People will search for their favorite database. I recommend that you do the last ones by databases:
- Silva, with in the description, the tools where you can find it
- GTDB, with in the description, the tools where you can find it
- ...